### PR TITLE
Fix missing partial crypto functionality

### PR DESCRIPTION
**Pull request by seba-aln**

### CODE REVIEW

File: src/PubNub/PubNub.php [hunk 1]
```diff
@@ -557,4 +559,18 @@ public function setToken($token)
...
+    public function getCrypto(): CryptoModule
+    {
+        if ($this->cryptoModule) {
+            return $this->cryptoModule;
+        } else {
+            return $this->configuration->getCrypto();
+        }
+    }
+
+    public function setCrypto(CryptoModule $cryptoModule)
```
Comment (marcin-cebo) [inline src/PubNub/PubNub.php:572]: FYI. In Kotlin/Java it is not possible to setCrypto in PubNub. User can setCrypto only in PNConfiguration.

**Review by marcin-cebo — APPROVED** [review]

File: src/PubNub/PubNub.php (hunk 1 above)
Comment (Xavrax) [reply src/PubNub/PubNub.php:572]: As I understood @techwritermat asked for that - maybe he will ask you too ;o

File: src/PubNub/PubNub.php [hunk 2]
```diff
@@ -557,4 +559,18 @@ public function setToken($token)
     {
         return $this->tokenManager->setToken($token);
     }
+
+    public function getCrypto(): CryptoModule
+    {
+        if ($this->cryptoModule) {
+            return $this->cryptoModule;
+        } else {
+            return $this->configuration->getCrypto();
```
Comment (Xavrax) [inline src/PubNub/PubNub.php:568]: Is this line creates a new cryptomodule on missing?

**Review by Xavrax — APPROVED** [review]

File: src/PubNub/PubNub.php (hunk 2 above)
Comment (seba-aln) [reply src/PubNub/PubNub.php:568]: if it has the cipher keyset up in config then will get the module. Else it raises an exception

**Pull request merged**